mp/src/game/client/hl2/hud_credits.cpp

```diff
@@ -319,7 +256,7 @@ void CHudCredits::DrawOutroCreditsName(void)
                         if (m_Alpha <= 0)
                         {
                             pCredit->bActive = false;
-                            engine->ClientCmd("creditsdone");
+                            CloseModal();
```

Comment:
You could probably also early return here, too. Saves rendering the panel while it closes?